server/src/main/java/eu/fasten/server/FastenServer.java

```diff
@@ -54,7 +51,17 @@
             defaultValue = ".")
     private Path pluginPath;
 
-    @Option(names = {"-t", "--topic"},
+    @Option(names = {"-s", "--skip_offsets"},
```

Comment:
Since this is a Kafka-related option, prefix with `k` -> so it becomes `-ks`.

Reply:
It's `-ks` now